src/test/java/io/vacco/mrx/MxConfigTest.java

```diff
@@ -44,6 +44,13 @@ public class MxConfigTest {
         var res3 = client.send(req, ofString());
```

Comment:
@zhaoyuango can you rename this variable to `res`? Thanks!

Reply:
thanks